src/datastation/scripts/retrieve_dataset_pids.py

```diff
@@ -6,23 +6,27 @@
 from datastation.dv_search import get_dataset_pids_from_search
 
 
-def retrieve_dataset_pids_command(config, dataverse_alias, output_filename):
-    logging.info("Retrieving dataset PIDs from URL {}, dataverse {}".format(config['dataverse']['server_url'], dataverse_alias))
-    pids = get_dataset_pids_from_search(config['dataverse']['server_url'], dataverse_alias)
-    # store in work dir, for further processing and or inspection
-    store_pids(pids, config['files']['output_dir'], output_filename)
+def retrieve_dataset_pids_command(server_url, dataverse_alias, output_dir, output_filename):
+    logging.info("Retrieving dataset PIDs from URL {}, dataverse {}".format(server_url, dataverse_alias))
+    pids = get_dataset_pids_from_search(server_url, dataverse_alias)
+    store_pids(pids, output_dir, output_filename)
 
 
 def main():
     config = init()
 
     parser = argparse.ArgumentParser(
         description='Retrieves the pids for all published datasets in the given collection')
-    parser.add_argument('-o', '--output', help='The output file, for storing the pids retrieved')
+    parser.add_argument('-o', '--output-dir', default='.', dest='output_dir',
```

Comment:
output dir now defaults to '.', the working dir, but previously it was the configured dir now in 'files_root'. I have open an issue for this last config change (DD-1083). Maybe we can get rid of that issue and fix it here, if we want to?